class AccountAlreadyExistException(Exception):
    pass


class AccountInvalidException(Exception):
    pass